README.md

```diff
@@ -292,11 +292,51 @@ extension that you created, and pass that into `StyleSheet.extend()`:
 ```js
 const mySelectorHandler = ...;
 
-const myExtension = { selectorHandler: mySelectorHandler };
+const myExtension = {selectorHandler: mySelectorHandler};
 
 StyleSheet.extend([myExtension]);
 ```
 
+As an example, you could write an extension which generates global styles like
+
+```js
+const globalSelectorHandler = (selector, _, generateSubtreeStyles) => {
+    if (selector[0] !== "*") {
+        return null;
+    }
+
+    return generateSubtreeStyles(selector.slice(1));
+};
+
+const globalExtension = {selectorHandler: globalSelectorHandler};
+```
+
+This might cause problems when two places try to generate styles for the same
+global selector however! For example, after
+
+```
+const styles = StyleSheet.create({
+    globals: {
+        '*div': {
+            color: 'red',
+        },
+    }
+});
+
+const styles2 = StyleSheet.create({
+    globals: {
+        '*div': {
+            color: 'blue',
+        },
+    },
+});
+
+css(styles.globals);
+css(styles2.globals);
+```
+
+It isn't determinate whether divs will be red or blue.
```

Comment:
Hrm, I'm not sure it's say it's non deterministic -- I think this falls into what compilers typically refer to as "undefined behavior", right? So we can say that it's undefined behavior whether the divs will be red or blue.